src/Form/MultiLinkField.php

```diff
@@ -138,7 +138,7 @@ private function loadFrom(DataObject $record): void
         }
 
         // Load ids from relation
-        $value = array_values($relation->getIDList() ?? []);
+        $value = $relation->sort(null)->column('ID');
```

Comment:
```suggestion
        $value = $relation->column('ID');
```
Sort order does matter here - it's what determines the order the links are displayed on the frontend

Reply:
Fixed and added a refresh to the behat to catch possible regressions of this in the future